libtonode-tests/tests/concrete.rs

```diff
@@ -1409,18 +1409,12 @@ mod slow {
             - first_send_to_transparent
             - (3 * u64::from(MARGINAL_FEE));
         assert_eq!(
-            recipient
-                .wallet
-                .shielded_balance::<OrchardDomain>(None, &[])
-                .await,
+            recipient.wallet.pending_balance::<OrchardDomain>().await,
             Some(expected_funds)
         );
         //  (2) The balance is not yet verified
         assert_eq!(
-            recipient
-                .wallet
-                .verified_balance::<OrchardDomain>(None)
-                .await,
+            recipient.wallet.confirmed_balance::<OrchardDomain>().await,
```

Comment:
These funds are the complementary (0) funds that *have* been confirmed at this point.   This assertion shows that no funds have yet been confirmed.

I propose that we move away from the term `verified` in favor of `confirmed` where `confirmed` is actually short-hand for a generally correct `confirmed_by_n_blocks` where `n == 1`.   So when we say `confirmed` we are using the shorthand of `confirmed_by_1_block` and if we mean confirmed by some other number of blocks, we say that number explicitly.